tests/AutoMapperTest.php

```diff
@@ -839,13 +839,11 @@ public function testAdderAndRemoverWithInstance(): void
         $this->autoMapper->map($petOwnerAsArray, $petOwner);
 
         self::assertIsArray($petOwner->getPets());
-        self::assertCount(3, $petOwner->getPets());
-        self::assertSame('Nemo', $petOwner->getPets()[0]->name);
-        self::assertSame('fish', $petOwner->getPets()[0]->type);
-        self::assertSame('Félix', $petOwner->getPets()[1]->name);
-        self::assertSame('cat', $petOwner->getPets()[1]->type);
-        self::assertSame('Coco', $petOwner->getPets()[2]->name);
-        self::assertSame('dog', $petOwner->getPets()[2]->type);
+        self::assertCount(2, $petOwner->getPets());
```

Comment:
poke @Korbeil is this acceptable ? should we consider this PR as a BC break given that i had to change that or do we consider this a bug ?

Reply:
I will merge this, we can always revert before release, i need this for another PR

Reply:
I do think it's fine since it's kinda a bug that we are adding without removing nowadays.